### PR TITLE
Add collaborators to MOTW

### PR DESCRIPTION
**Pull request by Pixelguymm**
I had to turn off all formatters I could think of, as formatting will create whitespace where there shouldn't be whitespace.

### CODE REVIEW

**Comment by TimothyGillespie** [comment]
I think this currently breaks because the map of the week list doesn't receive any collaborators from its `+page.server.ts`.

This would be solved if you push the collaborators in here:
https://github.com/beastsaber/bsaber/blob/4e42d2ac7e236069877d027807f812eb777e6c41/src/routes/maps-of-the-week/%5Bpage%5D/%2Bpage.server.ts#L46

And add them to the data with the lookup similar to the uploader.
Since every page holds 15 maps and the API allows a maximum of ids at once (and I don't have any logic that would allow handling more as of now) you could play it safe an easy by decreasing the pageSize to 10. That allows for 5 uploaders per map on average. That should be pretty safe.

https://github.com/beastsaber/bsaber/blob/4e42d2ac7e236069877d027807f812eb777e6c41/src/routes/maps-of-the-week/%5Bpage%5D/%2Bpage.server.ts#L4

File: src/routes/maps-of-the-week/[page]/+page.server.ts [hunk 1]
```diff
@@ -74,15 +74,18 @@ export async function load({ fetch, params }: LoadFunctionParameter): Promise<Ma
             map: {
                 id: singleMapOfTheWeek.mapId,
                 name: beatSaverMapData.name,
-                coverUrl: coverUrl,
+                coverUrl: coverUrl!!,
```
Comment (TimothyGillespie) [inline src/routes/maps-of-the-week/[page]/+page.server.ts:77]: What does post-variable `!!` do?
I don't think I know this language feature. I only know the non-null/undefined assertion with one `!`.

Comment (Pixelguymm) [reply src/routes/maps-of-the-week/[page]/+page.server.ts:77]: That might have been my Kotlin brain...
Anyway, I already fixed that in one of my unpushed commits.

**Comment by TimothyGillespie** [comment]
Looks good to me. The server side for map of the week is so much prettier now :laughing:

**Review by TimothyGillespie — APPROVED** [review]

**Pull request merged**